samples/eh_sender_with_sas_token_sample/main.c

```diff
@@ -205,7 +205,7 @@ int main(int argc, char** argv)
             {
                 for (i = 0; i < msg_count; i++)
                 {
-                    (void)messagesender_send(message_sender, message, on_message_send_complete, message);
+                    (void)messagesender_send_async(message_sender, message, on_message_send_complete, message, 0);
```

Comment:
>0 [](start = 111, length = 1)

Can you please use a timeout with one sample just to the user a way to use it.

Reply:
A wonderful idea. Done.